deps/wazuh_testing/wazuh_testing/qa_docs/lib/config.py

```diff
@@ -32,153 +36,194 @@ def __init__(self, *args):
             self.include_paths.append(os.path.join(self.project_path, path))
 
-    def _read_include_regex(self):
+    def __read_include_regex(self):
         """
         brief: Reads from the config file the regexes used to identify test files.
         """
+        Config.LOGGER.debug('Reading the regular expressions to include files from config file')
+
         if not 'Include regex' in self._config_data:
-            logging.error("Config include regex is empty")
-            raise Exception("Config include regex is empty")
+            raise QAValueError('Config include regex is empty', Config.LOGGER.error)
+
         self.include_regex = self._config_data['Include regex']
 
-    def _read_group_files(self):
+    def __read_group_files(self):
         """
         brief: Reads from the config file the file name to be identified with a group.
         """
+        Config.LOGGER.debug('Reading group files from config file')
+
         if not 'Group files' in self._config_data:
-            logging.error("Config group files is empty")
-            raise Exception("Config group files is empty")
+            raise QAValueError('Config include paths are empty', Config.LOGGER.error)
+
         self.group_files = self._config_data['Group files']
 
-    def _read_function_regex(self):
+    def __read_function_regex(self):
         """
         brief: Reads from the config file the regexes used to identify a test method.
         """
+        Config.LOGGER.debug('Reading the regular expressions to include test methods from config file')
+
         if not 'Function regex' in self._config_data:
-            logging.error("Config function regex is empty")
-            raise Exception("Config function regex is empty")
+            raise QAValueError('Config function regex is empty', Config.LOGGER.error)
+
         self.function_regex = self._config_data['Function regex']
 
-    def _read_ignore_paths(self):
+    def __read_ignore_paths(self):
         """
         brief: Reads from the config file all the paths to be excluded from the parsing.
         """
+        Config.LOGGER.debug('Reading the paths to be ignored from config file')
+
         if 'Ignore paths' in self._config_data:
             ignore_paths = self._config_data['Ignore paths']
+
             for path in ignore_paths:
                 self.ignore_paths.append(os.path.join(self.project_path, path))
 
-    def _read_module_fields(self):
+    def __read_module_fields(self):
         """
         brief: Reads from the config file the optional and mandatory fields for the test module.
         """
+        Config.LOGGER.debug('Reading mandatory and optional module fields from config file')
+
         if not 'Module' in self._config_data['Output fields']:
-            logging.error("Config output module fields is missing")
-            raise Exception("Config output module fields is missing")
+            raise QAValueError('Config output module fields is missing', Config.LOGGER.error)
```

Comment:
```suggestion
            raise QAValueError('Config output module fields are missing', Config.LOGGER.error)
```

Reply:
Done in: https://github.com/wazuh/wazuh-qa/commit/86e5045beb49f8e91dbb42b6e59f8d3164a57523